src/invenio-records-js/directives/invenioRecordsForms.js

```diff
@@ -171,7 +171,23 @@ function invenioRecordsForm($q, schemaFormDecorators, InvenioRecordsAPI,
           // Pre-process the query value/identifier
           query = options.scope.insideModel;
           query = scope.$eval(options.processQuery || 'query', {query: query});
+        } else if (Array.isArray(options.initial) && options.initial.length > 0) {
+          // If there is no query, no previous suggestions, no insideModel
```

Comment:
What is `insideModel`?

Reply:
It's one of the locations where the uiselect library saves data. I don't know why they couldn't use `internalModel`... This is why I was complaining a little last time: this library is using `internalModel`, `externalModel`, `insideModel`, `titleMap` and `model['<json key>']` as ways to store array of "terms" in various formats at various times. They had a reason, I don't know what it was.